components/stopwatch.rst

```diff
@@ -48,6 +48,18 @@ You can also provide a category name to an event::
 You can consider categories as a way of tagging events. For example, the
 Symfony Profiler tool uses categories to nicely color-code different events.
 
+Usage in symfony
+----------------
+
+In a Controller, do:
```

Comment:
This should be a double colon (`::`) in order to mark the next block as PHP code block.